src/SwipeableListItem.js

```diff
@@ -102,12 +102,28 @@ class SwipeableListItem extends PureComponent {
     requestAnimationFrame(this.updatePosition);
   };
 
-  handleTouchMove = event => this.handleMove(event, event.targetTouches[0]);
+  handleMouseMove = event => {
+    if (this.dragStartedWithinItem()) {
+      const { clientX, clientY } = event;
+
+      this.setDragDirection(clientX, clientY);
 
-  handleMouseMove = event => this.handleMove(event, event);
+      if (this.isSwiping()) {
+        event.stopPropagation();
+        event.preventDefault();
 
-  handleMove = (event, { clientX, clientY }) => {
+        const delta = clientX - this.dragStartPoint.x;
+        if (this.shouldMoveItem(delta)) {
```

Comment:
Consider spacing this out :)
```suggestion

        if (this.shouldMoveItem(delta)) {
```